cmd/console/console.go

```diff
@@ -172,7 +165,32 @@ func generateRespawnConf(cmdline string) []byte {
 
 	respawnConf.WriteString("/usr/sbin/sshd -D")
 
-	return respawnConf.Bytes()
+	return respawnConf.String()
+}
+
+func writeRespawn() error {
+	cmdline, err := ioutil.ReadFile("/proc/cmdline")
+	if err != nil {
+		return err
+	}
+
+	respawn := generateRespawnConf(string(cmdline))
+
+	files, err := ioutil.ReadDir("/etc/respawn.conf.d")
+	if err == nil {
+		for _, f := range files {
+			content, err := ioutil.ReadFile(f.Name())
+			if err != nil {
+				return err
+			}
+			respawn += fmt.Sprintf("\n%s", string(content))
+		}
+	} else if !os.IsNotExist(err) {
+		return err
```

Comment:
We probably don't want to return here. Still need to write `/etc/respawn.conf` even if there's an error reading `/etc/respawn.conf.d`.